protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -418,6 +418,40 @@ message RevertRepositoryCommitsRequest {
     }
 }
 
+message FindRepositories {
+    repeated RepositoryIdentification repository_id = 1;
+    repeated KeyValueQuery predicates = 2;
+    bool ids_only = 3; // whether to return full experiment runs or ids only
+
+    //For pagination
+    int32 page_number = 4;
+    int32 page_limit = 5;
+    bool ascending = 6;
+    string sort_key = 7;
+
+    message Response {
+        repeated Repository repository = 1;
+        int64 total_records = 2;
+    }
+}
+
+message FindRepositoriesBlobs {
+    repeated RepositoryIdentification repository_id = 1;
```

Comment:
add workspace, repo and commit as per the endpoint suggestion

Reply:
here just need to add `workspace_name` not the other fields

Reply:
Added

Reply:
I don't see the commit.

Reply:
@conradoverta The commit_hash should be single OR repeated?